rs/types/management_canister_types/src/lib.rs

```diff
@@ -2636,6 +2636,7 @@ pub struct SignWithSchnorrArgs {
     pub message: Vec<u8>,
     pub derivation_path: DerivationPath,
     pub key_id: SchnorrKeyId,
+    pub taproot_tree_root: Option<Vec<u8>>,
```

Comment:
I suppose you also need to add `#[serde(with = "serde_bytes")]` for this `Vec<u8>`. I'm not sure if it picks up the optimization when wrapped into `Option`.

Reply:
Done - now an `Option<ByteBuf>`